test-pipeline.yml

```diff
@@ -34,6 +34,10 @@ stages:
     files:
       - file: test-deployment.yml
       - file: test-configmap.yml
+    configuratorFiles:
+      - file: test-configurator2.yml
+        env: superOps
+      - file: test-configurator.yml
```

Comment:
write some tests for both of these situations and also one for if the pipeline doesn't have a configuratorfiles section